mbus/mbus-protocol.h

```diff
@@ -647,6 +647,9 @@ const char *mbus_unit_prefix(int exp);
 
 const char *mbus_data_error_lookup(int error);
 
+const char *mbus_vib_unit_lookup_FB(mbus_value_information_block *vib);
+const char *mbus_vib_unit_lookup_FD(mbus_value_information_block *vib);
```

Comment:
Why do you think this should be available to public?

Reply:
I cannot say I looked at this for very long time.
I am use to always write protypes, and I did not see a mbus-protocol-private.h where I would have put prototypes that should be considered private.

These does not have to be public. How do you normally do with such functions?

Reply:
In my opinion, the functions below should not be public, so I would argue that I have been following the current standard of the code.

```
const char *mbus_vib_unit_lookup(mbus_value_information_block *vib);
const char *mbus_vif_unit_lookup(unsigned char vif);
```

Reply:
Since these functions are only necessary for this file declare it static.